Kernel/Memory/PageDirectory.h

```diff
@@ -37,6 +37,15 @@ class PageDirectory : public RefCounted<PageDirectory> {
 #endif
     }
 
+    bool cr3_is_initialized() const
```

Comment:
The code follows the convention of `is_<blah>()`, we should probably do the same here.
```suggestion 
    bool is_cr3_initialized() const
```

Reply:
I was like 80% sure I had fixed that...